recipe/meta.yaml

```diff
@@ -53,3 +58,5 @@ about:
 extra:
   recipe-maintainers:
     - tduigou
+    - kenza12
+    
```

Comment:
This is not a real line ending as indicated by the red circle. This can be because of a setting in our editor.

To get a clean environment, I would recommend opening this on the Github website and adding one empty line there.

Reply:
Thanks @BastianZim

Reply:
Thank you @BastianZim , the linting succeeded but the checks keep being canceled. We tried to re-render but we got a permission error and the "Allow edits from maintainers" box is not available. Can you help us to re-run the checks please ? Thanks again.

Reply:
Have you checked if you enabled the box?

Reply:
I don't have any box to checked below the "Lock conversation" message. My account "tduigou" is not, as far as I understand, an organization account. I'm wondering if this could be due to that I am only "maintener" of the repository...

Reply:
No, it’s where the fork is based at. Not the account that is listed.

Reply:
OK thanks. So if I understand correctly: since this fork is based at "conda-forge", which is an organization, this means we cannot "tick the box", hence cannot rerender using the "@conda-forge-admin please rerender" option ?

Reply:
No, the fork and branch is brsynth:update_pysbol2_1_4. 
From what I can see, brsynth is an organization and therefore the bot cannot push to it.

You can also rerender locally though if you want to keep using this fork: https://conda-forge.org/docs/maintainer/updating_pkgs.html#dev-update-rerender

Reply:
OK thanks for the information. We finally managed to pass the commits through a new PR :)